suite-native/module-accounts-import/src/screens/XpubScanScreen.tsx

```diff
@@ -176,7 +173,7 @@ export const XpubScanScreen = ({
         <Screen
             header={<AccountImportScreenHeader closeActionType="back" />}
             footer={<XpubHint networkType={networkType} handleOpen={handleOpenHint} />}
-            extraKeyboardAvoidingViewHeight={EXTRA_KEYBOARD_AVOIDING_VIEW_HEIGHT}
+            focusedInputBottomOffset={105}
```

Comment:
```suggestion
            focusedInputBottomOffset={155}
```

To make the submit button visible even if the keyboard is still open.

Reply:
My testing show it has to be 163. Fixed.